plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/Group.php

```
@@ -0,0 +1,41 @@
+<?php
+/**
+ * WooCommerce Product Group Block class.
+ */
+
+namespace Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates;
+
+use Automattic\WooCommerce\Admin\BlockTemplates\BlockTemplateInterface;
+use Automattic\WooCommerce\Admin\BlockTemplates\ContainerInterface;
+use Automattic\WooCommerce\Internal\Admin\BlockTemplates\BlockContainerTrait;
+
+/**
+ * Class for Group block.
+ */
+class Group extends ProductBlock implements GroupInterface {
+	use BlockContainerTrait;
+
+	/**
+	 * Group Block constructor.
+	 *
+	 * @param array                   $config The block configuration.
+	 * @param BlockTemplateInterface  $root_template The block template that this block belongs to.
+	 * @param ContainerInterface|null $parent The parent block container.
+	 *
+	 * @throws \ValueError If the block configuration is invalid.
+	 * @throws \ValueError If the parent block container does not belong to the same template as the block.
+	 */
+	public function __construct( array $config, $root_template, ContainerInterface &$parent = null ) {
+		parent::__construct( array_merge( array( 'blockName' => 'woocommerce/product-tab' ), $config ), $root_template, $parent );
```

If a `blockName` is passed in, perhaps we should throw an exception (maybe `InvalidArgumentException`). Because, that should not be done, and the resulting behavior (ignoring the `blockName`) might not be clear to consumers.

---

Good suggestion, this got added.